xbmc/windowing/X11/WinSystemX11GLContext.cpp

```diff
@@ -233,3 +236,18 @@ std::unique_ptr<CVideoSync> CWinSystemX11GLContext::GetVideoSync(void *clock)
   }
   return pVSync;
 }
+
+void* CWinSystemX11GLContext::GetVaDisplay()
+{
+#if defined(HAVE_LIBVA)
+  if (dynamic_cast<CGLContextEGL*>(m_pGLContext))
```

Comment:
Ah OK it's temporarily disabled. So if you say that the check will be reenabled and that GetVaDisplay() will only be used with EGL I can remove it from X11.